topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,182 @@
+>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
+>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
+>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
+>
+>    This step applies polynomial curve fitting to model intensity fluctuations over time, identifying mitoflash events when intensity values exceed the fitted curve by the given significance level. Key parameters to analyze are **Amplitude** (peak intensity), **Tpk** (time to peak), and **T50** (duration). These values provide initial insights into mitochondrial function, with further interpretation outlined in the following section.
+>
+{: .hands_on}
+
+## Analysis and Interpretation of Results
+
+Finally, now you have your results table ready for analysis! The output is an Excel file containing the columns **FRAME**, **POS_X**, **POS_Y**, **INTENSITY**, **CURVE**, and **CURVE_A**. This information is available for all detected spots in your image stack. 
+
+With our data results, we can analyze their frequency, duration, and intensity. This data helps us understand the physiological significance of mitoflashes in processes like muscle contraction, neuron development, and wound healing.
+
+![Curve fitting results mitoflash](../../images/detection-of-mitoflashes/Curve_fitting_results_mitoflash.png "Mitoflash Curve Fitting Results: Data for plotting.")
+
+### Key Parameters for Analysis and Plotting
+
+1. **FRAME**: Represents time points in the data, with each row corresponding to a frame in the time-lapse sequence. Use **FRAME** as the x-axis for time-based plotting.
+
+2. **POS_X** and **POS_Y**: These columns provide positional coordinates of each detected mitoflash, allowing visualization of mitochondrial movement and spatial distribution within the cell.
+
+3. **INTENSITY**: Shows the raw intensity values for each mitochondrion at each time point. Plotting **INTENSITY** against **FRAME** reveals fluctuations over time, with peaks indicating mitoflash events.
+
+4. **CURVE** and **CURVE_A**: Contain fitted curve data that smooths out noise to highlight true intensity peaks. **CURVE** is the main fit, while **CURVE_A** may show adjusted fits if available.
+   
+   - **Amplitude (F/F0)**: The peak value in **CURVE** or **CURVE_A**, representing the magnitude of each mitoflash. Higher amplitudes indicate stronger bursts, which can have biological significance.
+   - **Tpk (Time to Peak)**: The frame where **CURVE** or **CURVE_A** reaches maximum intensity, indicating the speed of each mitoflash.
+   - **T50 (Duration)**: The frames where the curve values are at least half of the peak value, representing the mitoflash duration. Longer durations may reflect sustained mitochondrial activity.
```

Comment:
I think this needs to be adapted to be more consistent with the description of the curves above (see my comments for line 136). Where do those terms Amplitude, Tpk, T50 come from? I couldn't find them in the curve fitting tool.